data/json/items/melee/swords_and_blades.json

```
@@ -11,7 +42,8 @@
     "bashing": 6,
     "cutting": 18,
     "price_postapoc": 250,
-    "qualities": [ [ "BUTCHER", -50 ] ]
+    "qualities": [ [ "BUTCHER", -50 ] ],
+    "delete": { "flags": [ "STAB" ] }
```

```suggestion
    "qualities": [ [ "BUTCHER", -50 ] ]
```
`STAB` flag is not in neither documentation nor in `sword_wood`. i think this line is added as a mistake?

---

> Changed the 2-by-sword's damage from 12/1 to 4/10, and gave it stabbing damage instead of cutting. 

Double-check again, `sword_wood` was given the stag flag and its description emphasizes the whole "it's pointy" thing. See also why its stats are specially balanced with the pointy stick in mind.